wallet/src/account/mod.rs

```diff
@@ -301,75 +379,155 @@ impl Account {
         })
     }
 
+    fn mark_outputs_as_used<B: storage::Backend>(
```

Comment:
Used here probably doesn't make sense. Correct me if I'm wrong.

"Used" is for keys in the context of BIP-39, where a key is "used" if there's a transaction that it can spend on the blockchain.

"Spend", well... it's for outputs. We all know what it means.

I think there could be confusion in the naming here. Am I getting this right?

Reply:
Right. I renamed to `mark_output_as_seen`. I hope it's better.